### PR TITLE
KEP-4832: add SchedulerAsyncPreemption to the doc

### PR DESCRIPTION
**Pull request by sanposhiho**
### Description

Add KEP-4832 feature gate to the doc.

### Issue

KEP: https://github.com/kubernetes/enhancements/issues/4832

### CODE REVIEW

**Comment by tengqm** [comment]
Looks like the code has landed.
/approve

File: content/en/docs/reference/command-line-tools-reference/feature-gates/scheduler-async-preemption.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.32"
+---
+
+Running some expensive operation within [the scheduler's preemption](/docs/concepts/scheduling-eviction/pod-priority-preemption/) asynchronously,
+which improves the scheduling latency when the preemption involves in.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/scheduler-async-preemption.md:15]:
```suggestion
Enable running some expensive operations within the scheduler, associated with
[preemption](/docs/concepts/scheduling-eviction/pod-priority-preemption/), asynchronously.
Asynchronous processing of preemption improves overall Pod scheduling latency.
```

**Review by sftim — COMMENTED** [review]
Thanks

Try this rewording.

**Pull request merged**